decisions/wc_dark_cult_decisions.txt

```diff
@@ -363,8 +379,21 @@ decisions = {
 						owner = THIS
 						troops = {
 							archers = { 180 180 }
-							heavy_infantry = { 270 270 }
-							golems = { 50 50 }
+							thralls = { 500 500 }
+							light_infantry = { 150 150 }
+							heavy_infantry = { 300 300 }
+						}
+						attrition = 0
+						scaled_by_biggest_garrison = 1
+						reinforces = no
+						maintenance = no
+					}
+					spawn_unit = {
+						province = PREV
+						home = PREV
+						owner = THIS
+						troops = {
+							elite_infantry = { 50 50 }
```

Comment:
Same here

<details>

![image](https://user-images.githubusercontent.com/46576860/62796283-89928680-bae1-11e9-87d4-b6aa9e356b85.png)
![image](https://user-images.githubusercontent.com/46576860/62796232-67006d80-bae1-11e9-91da-cc0400445136.png)

</details>

Reply:
Is it the number of soldiers? To confirm

Reply:
If my math is correct, I should get only ~150 elite infantry from this decision, but for some reason I got 17 times more.
Maybe this effect included already spawned units in calculations, idk.